frontend/src/components/RegisterForm.tsx

```diff
@@ -166,7 +166,7 @@ const RegisterForm = () => {
                     helperText={
                       touched.email && errors.email
                         ? errors.email
```

Comment:
I saw a lot of repetitiveness like ` touched.email && errors.email` in every form field. I experienced little bit and found out how to make function that return true or false for given field (email, usernmae etc).
```
type fieldKeys = keyof typeof values
const checkTouchedAndErrors = (fieldName: fieldKeys): boolean =>
!!(touched[fieldName] && errors[fieldName])
```
If you wan't to make this change it can be used like this checkTouchedAndErrors('email')